bahmni_account/models/account_invoice.py

```diff
@@ -106,63 +87,10 @@ def action_post(self):
             other_moves._post(soft=False)
         return False
     
+class AccountPayment(models.Model):
+    _inherit = 'account.payment'
+    
+    def invoice_seach(self):
+        """ Using ref find the invoice obj """
+        return self.env['account.move'].search([('name', '=', self.move_id.ref),('move_type', '=', 'out_invoice')], limit=1)
    
-
-    @api.model
-    def _prepare_refund(self, invoice, date_invoice=None, date=None, description=None, journal_id=None):
```

Comment:
Is this not being used anywhere, and hence deleted?

Reply:
yes